client-js/prisma-models.d.ts

```diff
@@ -16,6 +16,7 @@ export type MemberActivity = {
   playerId: number;
   type: ActivityType;
   role?: GroupRole;
```

Comment:
I think technically these should be `GroupRole | null` instead of optional GroupRole 🤔  my bad on that

Reply:
fixed